test/event.js

```diff
@@ -0,0 +1,132 @@
+//During the test the env variable is set to test
+process.env.NODE_ENV = 'test';
+
+let mongoose = require("mongoose");
+let event = require('./models/event');
```

Comment:
When you load it correctly, you have an error in line 9 of `model/event.js` since the JSON is not well formed. Please, run `npm test` and correct the errors until it works properly. If you find some error you can't fix, just ask us ;)